languages/go/bitwarden_client.go

```diff
@@ -3,7 +3,7 @@ package sdk
 import (
 	"encoding/json"
 
-	"github.com/bitwarden/sdk/languages/go/internal/cinterface"
+	"github.com/bitwarden/sm-sdk-go/cinterface"
```

Comment:
The internal package was intentional, to limit the external APIs and reduce the surface for major versions. The `cinterface` should not be exposed externally. The reason this doesn't work is because you are trying to import it from the sm-sdk-go repository and it's loading two different packages.